sdk/translation/azure-ai-translation-text/azure/ai/translation/text/_version.py

```diff
@@ -6,4 +6,4 @@
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "1.0.0b2"
+VERSION = "1.0.0b1"
```

Comment:
The issue in ci if because of this change
```suggestion
VERSION = "1.0.0b2"
```